### PR TITLE
CT 111: Show STEM Filter and Indicator in Comparison Tool

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a Comparison Tool user, I need the ability to search for institutions that offer the Rogers STEM Scholarship and/or see if the institution offers the scholarship while viewing the profile page so that I can make informed decisions about how I choose to use my benefits.

Originating Issue:
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19177

## Testing done
Local manual, unit, and e2e tests passed

#### Screenshot 1:
Screen Shot 2019-07-16 at 12 36 34 PM

#### Screenshot 2:
Screen Shot 2019-07-16 at 12 36 58 PM

#### Screenshot 3:
Screen Shot 2019-07-16 at 12 37 06 PM

## Acceptance criteria
- [x] "Rogers STEM Scholarship" Filter is available to filter "Search Results" in the "Programs" section.
If the user uses the filter, only institutions that offer the Rogers STEM Scholarship (as indicated by the "STEM Indicator" column provided in the WEAMS.csv) AND meet all other search criteria.
- [x] The "Institution Summary" section on the "School Profile" page displays the following indicator for the Rogers STEM Scholarship: "Rogers STEM Scholarship:"
"Yes" is displayed if the institution offers the Rogers STEM Scholarship, as indicated by the "STEM Indicator" column provided in the WEAMS.csv.
"No" is displayed if the institution does not offer the Rogers STEM Scholarship, as indicated by the "STEM Indicator" column provided in the WEAMS.csv.
- [x] When the user clicks on "Rogers STEM Scholarship" a modal is displayed as outlined in story: #17904.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/applications/gi/components/profile/AdditionalInformation.jsx [hunk 1]
```diff
@@ -111,6 +111,19 @@ export class AdditionalInformation extends React.Component {
           &nbsp;
           {it.independentStudy ? 'Yes' : 'No'}
         </div>
+        <div>
```
Comment (zurbergram) [inline src/applications/gi/components/profile/AdditionalInformation.jsx]: does this need a production flag?

File: src/applications/gi/components/search/InstitutionFilterForm.jsx [hunk 2]
```diff
@@ -93,6 +93,12 @@ class InstitutionFilterForm extends React.Component {
           label="8 Keys to Vet Success"
           onChange={this.handleCheckboxChange}
         />
+        <Checkbox
```
Comment (zurbergram) [inline src/applications/gi/components/search/InstitutionFilterForm.jsx]: does this need a production flag?

File: src/applications/gi/components/profile/AdditionalInformation.jsx (hunk 1 above)
Comment (elder-joseph) [reply src/applications/gi/components/profile/AdditionalInformation.jsx]: Hey you are correct for both, adding the flags now

**Review by zurbergram — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]
Testing passes

**Comment by elder-joseph** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/gi/components/profile/AdditionalInformation.jsx [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
+import environment from '../../../../platform/utilities/environment';
```
Comment (rianfowler) [inline src/applications/gi/components/profile/AdditionalInformation.jsx]: we have a webpack resolve alias for the platform folder imports. relative path is fine but you can also just `import environment from 'platform/utilities/environment'` if you want

**Review by rianfowler — APPROVED** [review]
LGTM

File: src/applications/gi/components/profile/AdditionalInformation.jsx (hunk 3 above)
Comment (elder-joseph) [reply src/applications/gi/components/profile/AdditionalInformation.jsx]: Ah, I have seen that in the code actually, but got a warning in my IDE so I went with the the relative path. I may update that then merge.

**Pull request merged**